src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -175,21 +166,17 @@ def update_migrated_issue_alert(rule: Rule, data: UpdatedIssueAlertData):
     delete_workflow_actions(if_dcg=if_dcg)
     create_workflow_actions(if_dcg=if_dcg, actions=data["actions"])  # action(s) must exist
 
-    updated_data: dict[str, Any] = {
-        "name": data["name"],
-        "environment_id": data["environment"],
-        "enabled": True,
-        "owner_team_id": None,
-        "owner_user_id": None,
-        "config": {"frequency": data["frequency"] or Workflow.DEFAULT_FREQUENCY},
-    }
-    owner = data["owner"]
-    if owner:
-        if owner and owner.is_user:
-            updated_data["owner_user_id"] = owner.id
-        if owner and owner.is_team:
-            updated_data["owner_team_id"] = owner.id
-    workflow.update(**updated_data)
+    workflow.environment_id = rule.environment_id
+    if frequency := data["frequency"]:
+        workflow.config["frequency"] = frequency
```

Comment:
Do we want to set this to the `Workflow.DEFAULT_FREQUENCY` otherwise?

Reply:
currently this looks like one of the fields where we only update it if it's not `None`, so i mimicked that behavior

https://github.com/getsentry/sentry/blob/11b5c06d25545f25c9f1174bc8fc15139f886ba7/src/sentry/projects/project_rules/updater.py#L69-L71